TTHAnalysis/python/tools/higgsDiffRecoTTH.py

```diff
@@ -141,7 +146,7 @@ def analyze(self, event):
                     # Optionally, later constrain the mass to the W PDG mass (although we use this only to build the candidate, and we store it without constraint)
                     W = j1+j2
                     mW = W.M()
-                    if mW<self.cuts_mW_had[0] or mW>self.cuts_mW_had[1]: continue
+#                    if mW<self.cuts_mW_had[0] or mW>self.cuts_mW_had[1]: continue
```

Comment:
Is this the latest decision we made? If it's just commented because of a test, please uncomment it again, otherwise it's fine as it is.